projects/scripts/adi_project.tcl

```diff
@@ -212,14 +212,19 @@ proc adi_project_run {project_name} {
 
   file mkdir $project_name.sdk
 
-  if [expr [get_property SLACK [get_timing_paths]] < 0] {
-    file copy -force $project_name.runs/impl_1/system_top.sysdef $project_name.sdk/system_top_bad_timing.hdf
-  } else {
+  set timing_paths_list [get_property -quiet SLACK [get_timing_paths -quiet]]
```

Comment:
This command never going to return a list of values, it will return the WNS. (Worst Negative Slack) So the variable name is a little bit misleading.

Reply:
done